daprdocs/layouts/partials/hooks/body-end.html

```diff
@@ -5,7 +5,7 @@
     // Your apiKey and indexName will be given to you once
     // we create your config
     apiKey: '54ae43aa28ce8f00c54c8d5f544d29b9',
-    indexName: 'crawler_dapr-rc3',
+    indexName: 'crawler_dapr',
```

Comment:
@AaronCrawfis - I'm assuming this doesn't effect anything right now since we don't have a v1.1 website. If we create one, does this need to change?

Reply:
We can keep it and fix it later. Doesn't hurt anything